src/FNA3D_Driver_OpenGL.c

```diff
@@ -106,6 +106,7 @@ struct OpenGLRenderbuffer /* Cast from FNA3D_Renderbuffer* */
 {
 	GLuint handle;
 	OpenGLRenderbuffer *next; /* linked list */
+	FNA3D_SurfaceFormat format;
```

Comment:
Link needs to be at the bottom

```suggestion
	FNA3D_SurfaceFormat format;
	OpenGLRenderbuffer *next; /* linked list */
```